editor/app/src/main/java/ca/ubc/eml/soiltopargraphy/editor/ui/main/MainFragment.kt

```diff
@@ -29,6 +40,9 @@ class MainFragment : Fragment(), OnMapReadyCallback, GoogleMap.OnMapLongClickLis
     //Google maps
     lateinit var googleMap: GoogleMap
     lateinit var mMapView: MapView
+    val coordinates = LatLng(50.713836, -120.350008)
+    var appRepo = AppRepository(Application())
```

Comment:
Instead of creating an instance of `Application`, use the singleton by getting [`activity.application`](https://developer.android.com/reference/android/app/Activity#getApplication())